testingenv/centos8/docker-entrypoint.sh

```
@@ -0,0 +1,25 @@
+#!/bin/bash
```

I'm wondering why the build didn't fail considering those issues with `rpm` dependencies.

I think we'll need the following:
```suggestion
#!/bin/bash

set -eu
set -o pipefail
```
in all `docker-entrypoint.sh` files to make sure we fail earlier.

We could easily miss this failure if not reading the logs..

---

Would be great to add it to all `docker-entrypoint.sh` as well, not only EL8 to make sure we don't step on the same rake twice.

---

Still need similar behavior to fail fast for other platforms (not only EL8).

---

Let’s do that In a separate PR. Will create an issue.

---

Sounds even better :+1:

---

https://github.com/StackStorm/st2chatops/issues/143